Decred Wallet/Features/Security/SecurityViewController.swift

```diff
@@ -67,22 +69,22 @@ class SecurityViewController: SecurityBaseViewController {
     }
     
     func activatePasswordTab() {
-        tabController?.selectedIndex = 0
+        self.tabController?.selectedIndex = 0
```

Comment:
We could use an enum of rawValue `Int` for the tabBarController Indexes. As such it would be more readable for instance, `tabController?.selectedIndex = enum.rawValue`.

Reply:
There are just 2 tabs, indices 0 and 1. We can _probably_ do without enums.